src/cryptojwt/jwt.py

```diff
@@ -88,7 +88,7 @@ def __init__(
         sign_alg="RS256",
         encrypt=False,
         enc_enc="A128CBC-HS256",
```

Comment:
@jschlyter can we put `A128GCM` instead of `A128CBC-HS256`

did you mean this?

Reply:
Done here
https://github.com/IdentityPython/JWTConnect-Python-CryptoJWT/pull/125/commits/619f9ef2ed60b6e95af02c9269a98e09922ed834